### PR TITLE
Export form behavior

### PR DESCRIPTION
**Pull request by homing1**
Task: task-[680557](https://dev.azure.com/ceapex/Engineering/_workitems/edit/680557), [680541](https://dev.azure.com/ceapex/Engineering/_workitems/edit/680541/), [680544](https://dev.azure.com/ceapex/Engineering/_workitems/edit/680544/)

Link: preview-430

This PR allows a few form behavior items for export and adds a test for when form is submitted without edits. It also removes star rating and two tests for form behavior. 

## Testing

1. Visit https://design.docs.microsoft.com/pulls/430/components/form.html . You should no longer see star rating in the form behavior section.

### CODE REVIEW

File: site/src/components/form.md [hunk 1]
```diff
@@ -172,28 +172,11 @@ The form behavior component can accept certain HTML attributes.
 				minlength="10"
 				rows="4"
 				cols="30"
+				required
```
Comment (ollips) [inline site/src/components/form.md:175]: I see the `textarea` was marked as required but when I'm trying to submit a form it doesn't highlight it with red border or anything

image
 
And it passes the empty submission

image

Comment (ollips) [reply site/src/components/form.md:175]: Oh, I didn't notice that initially but it looks like the textarea contains some empty spaces by default

Comment (homing1) [reply site/src/components/form.md:175]: Made an update to remove the empty spaces

**Review by ollips — DISMISSED** [review]

File: integration/tests/form-behavior.spec.ts [hunk 2]
```diff
@@ -39,37 +39,6 @@ test.describe('form behavior validation', () => {
 		});
 	});
 
-	// The input value string is automatically reduced if input has maxlength
```
Comment (homing1) [inline integration/tests/form-behavior.spec.ts:42]: For a normal input, setting the `maxlength` attribute will automatically limit the number of characters to its value so this test is redundant.

**Review by jdanyow — APPROVED** [review]

File: js/src/elements/form-behavior.ts [hunk 3]
```diff
@@ -358,7 +356,7 @@ class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
-			if (input.hasAttribute('aria-hidden')) {
+			if (input.hasAttribute('aria-hidden') === true) {
```
Comment (wibjorn) [inline js/src/elements/form-behavior.ts:359]: curious why this change was necessary, seems like same with but with more characters

Comment (homing1) [reply js/src/elements/form-behavior.ts:359]: In an earlier PR, Ben made a comment on how `aria-hidden="false"` would trip this up. Perhaps it's an unlikely scenario to explicit set the attribute to false, but I suppose it can still happen.

File: .changeset/sweet-walls-sing.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+'@microsoft/atlas-integration': minor
```
Comment (wibjorn) [inline .changeset/sweet-walls-sing.md:2]: patch patch patch

Comment (wibjorn) [reply .changeset/sweet-walls-sing.md:2]: nvm, other minors changes are coming in so we can ignore

**Review by wibjorn — APPROVED** [review]
